bindings/python/manifold3d.cpp

```diff
@@ -315,6 +315,14 @@ NB_MODULE(manifold3d, m) {
       .def("calculate_curvature", &Manifold::CalculateCurvature,
            nb::arg("gaussian_idx"), nb::arg("mean_idx"),
            manifold__calculate_curvature__gaussian_idx__mean_idx)
+      .def(
+          "min_gap",
+          [](const Manifold &self, const Manifold &other, float searchLength) {
+            return self.MinGap(other, searchLength);
+          },
```

Comment:
Is this function necessary, or can you replace it with `&Manifold::MinGap`?

Reply:
Oh right, looks like that works as well